# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/canvas/ClientWebGLContext.cpp
# Commit: 75ddac3e4f78
# Full Hash: 75ddac3e4f787f72e2b737a65353329c1e0ce0ec
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-29 21:37:21
# Description:
#   Bug 1608330 - Use Run<RPROC> instead of calling Present directly. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60091
# ==============================================================================

diff -r 2d73dd4ab498 -r 75ddac3e4f78 dom/canvas/ClientWebGLContext.cpp
--- a/dom/canvas/ClientWebGLContext.cpp	Wed Jan 29 08:29:13 2020 +0000
+++ b/dom/canvas/ClientWebGLContext.cpp	Thu Jan 16 03:09:13 2020 +0000
@@ -439,7 +439,7 @@
   if (!mNotLost) return;
   if (mNotLost->inProcess) {
     WEBGL_BRIDGE_LOGI("[%p] Presenting", this);
-    mNotLost->inProcess->Present();
+    Run<RPROC(Present)>();
   }
 }
 
